x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/SearchableSnapshotActionIT.java

```diff
@@ -574,6 +572,113 @@ public void testConvertingSearchableSnapshotFromFullToPartial() throws Exception
         );
     }
 
+    @SuppressWarnings("unchecked")
+    public void testResumingSearchableSnapshotFromFullToPartial() throws Exception {
```

Comment:
Should we also add a test for the other use case this fixes - `An ILM policy with the a searchable snapshot action in hot or cold is added on a partially mounted searchable snapshot.` ?

Reply:
Good idea, I thought it is more or less captured but it's better to be thorough